src/FieldVector.jl

```diff
@@ -23,8 +23,8 @@ abstract type FieldVector{N, T} <: StaticVector{N, T} end
 @propagate_inbounds setindex!(v::FieldVector, x, i::Int) = setfield!(v, i, x)
 
 # See #53
-Base.cconvert{T}(::Type{Ptr{T}}, v::FieldVector) = Ref(v)
-Base.unsafe_convert{T, FV <: FieldVector}(::Type{Ptr{T}}, m::Ref{FV}) =
+Base.cconvert{T}(::Type{Ptr{T}}, v::FieldVector) = Base.RefValue(v)
+Base.unsafe_convert{T, FV <: FieldVector}(::Type{Ptr{T}}, m::Base.RefValue{FV}) =
     _unsafe_convert(Ptr{T}, eltype(FV), m)
-_unsafe_convert{T, FV <: FieldVector}(::Type{Ptr{T}}, ::Type{T}, m::Ref{FV}) =
+_unsafe_convert{T, FV <: FieldVector}(::Type{Ptr{T}}, ::Type{T}, m::Base.RefValue{FV}) =
          Ptr{T}(Base.unsafe_convert(Ptr{FV}, m))
```

Comment:
Sorry, not primitive type.  Broken when `T` is some kind of `mutable struct`?

A `FieldVector` of these `T` is laid out as an array of pointers

```C
struct FV
{
    T* data[N];
};
```

Then creating a `RefValue{FV}` basically gives a `FV*`, which is a `T**`, which can't be meaningfully converted to a `T*` using `Ptr{T}`.  (Sorry for the mix of C and julia notation)